src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/models/Address.java

```diff
@@ -15,4 +19,12 @@
     private List<String> addressLines;
     private String postcode;
     private String town;
+
+    @Override
+    public String toString() {
+        return Stream.of(addressLines, Arrays.asList(postcode, town))
+            .flatMap(Collection::stream)
+            .map(s -> s.replaceAll("\\r\\n|\\r|\\n", ""))
```

Comment:
`"\\r\\n|\\r|\\n"` is equivalent to `"[\\n\\r]"`

Reply:
that's what I had before 🤦‍♀

Reply:
will fix when I rebase 👍

Reply:
FYI This regex will go away in another story in which we map address directly to CCD address, so we won't need to convert to string anymore :D